sycl/source/detail/scheduler/commands.cpp

```diff
@@ -239,8 +239,20 @@ void Command::waitForEvents(QueueImplPtr Queue,
   if (!EventImpls.empty()) {
     std::vector<RT::PiEvent> RawEvents = getPiEvents(EventImpls);
```

Comment:
The RawEvents initialized here is not used if Queue->is_host is true.
Please move it to the line 257

Reply:
Done